enterprise-suite/gotests/util/util.go

```diff
@@ -195,11 +194,11 @@ const firstSleepMs = 20
 const maxSleepMs = 10000
 
 // Repeatedly runs a function, sleeping for a bit after each time, until it returns true or reaches maxRepeats.
-// failMsg will be printed in the result error in case of timeout, can be empty.
-func WaitUntilTrue(f func() bool, failMsg string) error {
+func WaitUntilTrue(f func() error) error {
```

Comment:
I think this should be named WaitUntilSuccess now. Also, the comment needs updating.

Reply:
K, will do.